File: CS50 problem sets/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    
    do 
    {
        n = get_int("Enter the desired hight:  "); // ask the user for the hight
    }
    while ((n > 8) || (n < 1)); // logical condition 
    
    for (int i = 0; i < n ; i++)
    {
        // printing the spaces
        for (int x = 0 ; x < (n - i - 1) ; x++)
        {
            printf(" ");
        }
        // printing the boxex
        for (int j = 0 ; j <= i ; j++)
        {
            printf("#");
        }
        //go to new line each time
        printf("\n");
    }
    
}

File: CS50 problem sets/pset6/mario/mario.py
from cs50 import get_int
n = get_int("Height: ")
while n < 1 or n > 8:
    n = get_int("Height: ")
for x in range(n):
    for i in range(0, n - x - 1, 1):
        print(" ", end="")
    for j in range(x + 1):
        print("#", end="")
    print()
